subjects/DPxAI/listed/README.md

```diff
@@ -0,0 +1,149 @@
+## Listed
+
+> Mindful AI mode
+
+### Context
+
+Sometimes we don't need a key, we just want a list of things, JS has a special type for that, called an array.
+
+In JavaScript, arrays are essential tools for handling these lists efficiently.
+
+Let's discover them together!
+
+### AI-Powered Learning Techniques
+
+`Example-Based Learning Technique:`
+This type of prompt encourages the AI to provide concrete examples to illustrate concepts, making it easier to understand and apply them.
+
+Find the examples across the subject ;)
+
+### Concepts
+
+### Understanding Arrays
+
+Arrays are special types of objects in JavaScript used to store lists of items. Unlike objects, arrays don't have keys for each element, just a list of values.
+
+**Example of an Array**
+
+Let's see an example of an array:
+
+```js
+let batteryLevels = [
+  80, // <- no keys!
+  60,
+  90,
+  50,
+];
+
+// or for brevity, we often write them on a single line like so:
+
+let batteryLevels = [80, 60, 90, 50];
+```
+
+### Indexes in Arrays
+
+The position of an element in an array is called its index, starting from 0. So, our batteryLevels array is roughly equivalent to writing this object:
```

Comment:
For consistency, `batteryLevels` should be formatted as the other variables in the subject

Reply:
Changed.